tests/layer_tests/mo_python_api_tests/test_mo_convert_complex_params.py

```diff
@@ -4,7 +4,9 @@
 import numpy as np
 import os
 import pytest
-from openvino.runtime import Model, Layout, PartialShape, Shape, layout_helpers, Type, Dimension, InputCutInfo, LayoutMap
+from openvino.runtime import Model, Layout, PartialShape, Shape, layout_helpers, Type, Dimension
+from openvino.tools.ovc import InputCutInfo
```

Comment:
It's better to take it from `mo`, because in `ovc` we are going to remove it. Postpone to the next PR, don't do in this one.

Reply:
Why?)